src/middlewares/yup.middlewares.ts

```diff
@@ -5,7 +5,7 @@ import { AnyYupSchema } from "../helpers/validation";
 export const validation =
   (schema: AnyYupSchema) =>
   async (req: Request, res: Response, next: NextFunction) => {
-    const resource = req.body;
+    const resource = req.body && req.query;
```

Comment:
it's really strange 0_o. What happen if you'll provide body and query in the same time?

please validate full `req` object with one single schema, with next keys:
```
{
  body: ...
  query: ...
}
```